### ISSUE TITLE
Consider merge with other existing original Xbox extensions

### ISSUE THREAD

**Issue by JayFoxRox** [jonas-schievink/GhidraXBE#1]
I also have an XBE loader at https://github.com/JayFoxRox/ghidra-xbox-extensions

Unfortunately both extensions are very similar, and each of them is hacked together in java.
Ideally we shouldn't have redundant code.

One of those projects should be moved to the XboxDev org (https://github.com/XboxDev/XboxDev) in the future as it's essential tooling, too.

There are also plans to integrate https://github.com/Cxbx-Reloaded/XbSymbolDatabase as part of an analyzer extension. Would be cool if you could join XboxDev communication channels to discuss these things (#xqemu, as you are on freenode anyway, or Discord). Very cool to see someone else working in this ecosystem.

**Comment by jonas-schievink** [comment]
I agree it'd be nice to reduce the duplication here, but unfortunately this is about as far as my Java-foo goes, so I don't think I'd be a good maintainer for this. I'll definitely try to (poorly) review and merge PRs, but since I'm not really involved with Xbox RE anymore I probably won't put too much time into this project myself.
